manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Harmonoid.Harmonoid
+PackageVersion: 0.2.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x64
+ReleaseDate: 2022-05-07
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/harmonoid/harmonoid/releases/download/v0.2.3/harmonoid-windows-setup.exe
+  InstallerSha256: 74536AC6FD82A75B467584F462B6265028FF9A120D6E39D533025DE480DE83C4
```

Comment:
May you please update this to `19045770E06B81A00BB9D50B113FA521A32BBD791139697DE4F569464D7F1B15`.

Thanks!